backend-service/src/main/java/com/smartparkinglot/backendservice/controller/DriverController.java

```diff
@@ -37,13 +39,15 @@
         return new ResponseEntity(driverService.Login(email, password),HttpStatus.ACCEPTED);
     }
 
-    @DeleteMapping()
+    @DeleteMapping("/delete")
+    @Operation(summary = "use it for driver deletion")
```

Comment:
Swagger documentations purpose is not like this. I mean we are using swagger for our API's docs. So you should say like `delete` endpoint deletes driver user. Not "use it for driver deletion".